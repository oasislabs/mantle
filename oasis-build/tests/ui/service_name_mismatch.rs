#[derive(oasis_std::Service)]
pub struct Service;

fn main() {
    oasis_std::service!(Serivce);
}


