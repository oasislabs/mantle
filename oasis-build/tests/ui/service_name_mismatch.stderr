error: Could not find state struct for `Serivce`
 --> $DIR/service_name_mismatch.rs:5:5
  |
5 |     oasis_std::service!(Serivce);
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^

error: aborting due to previous error

